src/styles/sidebar/components/menu-item.scss

```diff
@@ -124,6 +117,4 @@ $menu-item-padding: 10px;
     width: 12px;
     height: 12px;
   }
-
-  @include outline-on-keyboard-focus;
```

Comment:
The mixin referenced here was missing. I will add this feature in a separate PR to keep the diff in this PR smaller.